client/src/app/pages/main/header/header.component.scss

```diff
@@ -5,10 +5,10 @@
   display: flex;
   width: 100%;
   height: 100vh;
-  min-height: 35rem;
-  padding: 15rem 0;
+  min-height: 55rem;
   z-index: 0;
   overflow: hidden;
+  background: rgba(22, 22, 22);
```

Comment:
Does an `rgba` without an alpha value default to 1.0? If so, could this be hexcode instead?

Reply:
Yes it does! I'll change it to hex and push the commit.